.github/workflows/consistency-functional.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           --health-timeout=5s
           --health-retries=3
     steps:
-      - uses: actions/checkout@v1
```

Comment:
We should change both:

https://github.com/os-migrate/os-migrate/pull/137/files#diff-37c7821a4a9822799e5e8cfcd0b9d699R21

Reply:
Sorry but the link does not seem to work, we have also a call to the checkout@v1 action in line 21